### ISSUE TITLE
Improve test suite speed by storing PNG versions of test fixtures

### ISSUE THREAD

**Issue by jnothman** [camelot-dev/camelot#47]
Currently lots of tests call `read_pdf`. This wastes time converting the PDF to raster images. This could be avoided.

**Comment by vinayak-mehta** [comment]
You're right, the PDF conversion for every test is not necessary. This looks like an easy enhancement, let me look into this over the weekend.

**Comment by bosd** [comment]
Closing as part of a housekeeping pass — this enhancement request is 4+ years old (filed in the atlanhq/camelot era), has no traction, and the architectural changes since (playa-pdf migration, the new \`network\`/\`hybrid\` parsers, the parallel + cpu_count work in #712) have moved the surrounding code substantially. If this is still a concern on **current master**, a focused fresh issue with a reproducer / concrete use-case would be much more actionable than rebasing this one. Reopen if you disagree.
